llvm/include/llvm/Support/GenericLoopInfoImpl.h

```diff
@@ -159,6 +159,17 @@ BlockT *LoopBase<BlockT, LoopT>::getUniqueExitBlock() const {
   return getExitBlockHelper(this, true).first;
 }
 
+template <class BlockT, class LoopT>
+BlockT *LoopBase<BlockT, LoopT>::getUniqueLatchExitBlock() const {
+  BlockT *Latch = getLoopLatch();
+  assert(Latch && "Latch block must exists");
+  auto IsExitBlock = [&](BlockT *BB, bool AllowRepeats) -> BlockT * {
```

Comment:
nit: sufficient to capture `this`?
```suggestion
  auto IsExitBlock = [this](BlockT *BB, bool AllowRepeats) -> BlockT * {
```